### PR TITLE
Add tab index and adjust image alts

### LINKED ISSUE #44: Accessibility - tabbing
You must be able to tab through your app and use it without a mouse

### LINKED ISSUE #48: Accessibility - semantic HTML
Your HTML must be written semantically and ARIA tags should be used (ONLY if needed / appropriate)

### PR DESCRIPTION
**Pull request by CalebCyphers**
## Description

- Adjust HTML for accessibility
- Add new alts and adjust old alts for images on the page
- add tab-indexes to elements

## Affected areas of application

- index.html
- scripts.js

## How Has This Been Tested?

- [x] Chrome browser

## Relevant Tickets

What tickets/issues were resolved by this feature

- Closes #44 
- Closes #48

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -23,17 +23,16 @@ let domUpdates = {
       if (favorites !== undefined && favorites.includes(String(recipe.id) )) {
          src = "https://image.flaticon.com/icons/svg/148/148841.svg"
       }
-    recipeCards.innerHTML += `<article id = ${recipe.id} class="single-recipe-card ">
+    recipeCards.innerHTML += `<article id = ${recipe.id} class="single-recipe-card">
          <article class="all-card-icons">
-           <img class="plus-icon card-icon" src="./images/plus-icon.png" alt="plus icon used to expand and show recipe details">
-           <img class="star-icon card-icon" src=${src}
-          "alt="empty star icon used to favorite and unfavorite recipes">
+           <img class="plus-icon card-icon" tabindex="0" src="./images/plus-icon.png" alt="show recipe details button">
```
Comment (nicolegooden) [inline src/domUpdates.js:28]: nice job adding the `tabIndex` attribute for accessibility!

File: src/domUpdates.js [hunk 2]
```diff
@@ -81,12 +80,12 @@ let domUpdates = {
...
       pantryArea.innerHTML += `<article class="single-ingredient-card">
     <article class="ingredient-name-area">
-      <h2 class="ingredient-name">${ingInPantry.name}</h2>
+      <h2 class="ingredient-name" tabindex="0">${ingInPantry.name}</h2>
     </article>
     <article class="ingredient-quantity-section">
-      <button>-</button>
-      <p class = 'ingredient-quantity'>${ingInPantry.amount}</p>
-      <button>+</button>
+      <button alt="decrement ingredient quantity button">-</button>
```
Comment (nicolegooden) [inline src/domUpdates.js:86]: thanks for adding those `alt` tags for the buttons, we were definitely overlooking that aspect.

File: src/index.html [hunk 3]
```diff
@@ -11,33 +11,33 @@
...
-        <h2 class="menu-my-pantry-title"><a href="./pantry.html">My Pantry</a></h2>
+        <h2 class="menu-my-pantry-title" tabindex="0"><a href="./pantry.html">My Pantry</a></h2>
       </article>
       <article class="menu-my-upcoming-recipes menu-selection hidden">
-        <h3 class="menu-my-upcoming-recipes-title">My Upcoming Recipes</h2>
+        <h3 class="menu-my-upcoming-recipes-title" tabindex="0">My Upcoming Recipes</h2>
       </article>
       <article class="menu-favorites menu-selection hidden">
-        <h3 class="view-favorites">Favorites</h2>
+        <h3 class="view-favorites" tabindex="0">Favorites</h2>
```
Comment (taylorjohnson141) [inline src/index.html:30]: Great job on the tab index

File: src/domUpdates.js [hunk 4]
```diff
@@ -81,12 +80,12 @@ let domUpdates = {
...
+      <p class = 'ingredient-quantity' tabindex="0">${ingInPantry.amount}</p>
+      <button alt="incriment ingredient quantity button">+</button>
```
Comment (taylorjohnson141) [inline src/domUpdates.js:88]: Thank you for putting in all the alts!

**Review by taylorjohnson141 — APPROVED** [review]

**Review by nicolegooden — APPROVED** [review]

**Pull request merged**